addons/auth_password_policy/static/src/password_meter.js

```diff
@@ -26,4 +30,18 @@ export class Meter extends Component {
     get value() {
         return computeScore(this.props.password, this.props.required, this.props.recommended);
     }
+
+    get strengthTextParams() {
```

Comment:
I agree keeping the functions arranged alphabetically would help in locating a specific function, but IMO we should always add the new code below the existing code, so one can better track the changes.

Secondly, in our case it was a single function, for once if we assume we add multiple functions, then it would become really difficult to track changes made to a file.

Here is an example where the newly added getter method is not present alphabetically but present at the last.
https://github.com/odoo/enterprise/blame/master/social/static/src/js/stream_post_comments_reply.js#L220

However, if you still feel it should be alphabetically, I would have no problems doing it. :+1:

Reply:
Place functions alphabetically. (Starting something is always the right choice. )